docs/src/dev/understanding_traces.md

```diff
@@ -15,7 +15,7 @@ to print IR to stderr.
 
 The following stages are supported:
 
- - `aot`: the AOT IR embedded in the ahead-of-time compiled binary.
+ - `aot`: the AOT IR trace.
```

Comment:
I'm not sure about this one. When you print the AOT IR, you really do print a CFG.

Reply:
I don't think "CFG" is a useful discriminator as all the IRs can be classified as CFGs. Do you mean "When you print the AOT IR you print the AOT IR for the entire C program" or ... ?

Reply:
Yes. This documentation is describing what IR phases you can log. When you log `aot` it's not a trace, its the whole interpreter's worth of IR, branches and all.

`jit-pre-opt` and `jit-post-opt` are (unintuitively)  AOT IR traces though.

Do you follow?

Reply:
Does 91034b6f8e3a4450466a48475e5acc0ecac967e1 fix things?

Reply:
Yes, thanks!